<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1163</width>
    <height>786</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>1131</width>
      <height>711</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Ejercicio 1</string>
     </attribute>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>171</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>50</y>
        <width>171</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="agregarEstudiante">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>120</y>
        <width>171</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Agregar Estudiante</string>
      </property>
     </widget>
     <widget class="QPushButton" name="imprimiEstudiantes">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>100</y>
        <width>171</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Imprimir</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>180</y>
        <width>401</width>
        <height>231</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="guardarEnArchivo">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>480</y>
        <width>171</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Guardar en archivo</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>20</y>
        <width>231</width>
        <height>28</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Estudiante con Licenciatura</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Estudiante con Maestria</string>
       </property>
      </item>
     </widget>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>230</y>
        <width>171</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_4">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>270</y>
        <width>171</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="agregarMateria">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>270</y>
        <width>171</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Agregar Materia</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_5">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>340</y>
        <width>171</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Nombre:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>20</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Edad:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>200</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Nombre:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>240</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Materia:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>310</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Nota:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Ejercicio 2</string>
     </attribute>
     <widget class="QLineEdit" name="lineEdit_6">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>60</y>
        <width>171</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_7">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>60</y>
        <width>171</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="agregarVehiculo">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>120</y>
        <width>171</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Agregar Vehiculo</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_8">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>60</y>
        <width>171</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_2">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>140</y>
        <width>91</width>
        <height>28</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Camion</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Auto</string>
       </property>
      </item>
     </widget>
     <widget class="QLineEdit" name="lineEdit_9">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>60</y>
        <width>171</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="imprimiEstudiantes_2">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>250</y>
        <width>171</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Imprimir</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_2">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>330</y>
        <width>401</width>
        <height>231</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="guardarEnArchivo_2">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>460</y>
        <width>171</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Guardar en archivo</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>30</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Modelo</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>30</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Year:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>30</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Kilometraje:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>30</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Peso:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Ejercicio 3</string>
     </attribute>
     <widget class="QPushButton" name="agregarEmpleado">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>100</y>
        <width>171</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Agregar Empleado</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_10">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>40</y>
        <width>171</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_11">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>40</y>
        <width>171</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_12">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>40</y>
        <width>171</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_13">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>171</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_3">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>120</y>
        <width>141</width>
        <height>28</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Contratado</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Fijo</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="guardarEnArchivo_3">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>460</y>
        <width>171</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Guardar en archivo</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>330</y>
        <width>401</width>
        <height>231</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="imprimiEstudiantes_3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>250</y>
        <width>171</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Imprimir</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>10</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Nombre:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>10</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Edad:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>10</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Salario:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>10</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Horas:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Ejercicio 4</string>
     </attribute>
     <widget class="QLineEdit" name="lineEdit_14">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>171</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_15">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>40</y>
        <width>171</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_4">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>110</y>
        <width>171</width>
        <height>28</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Libro</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Revista</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Articulo Cientifico</string>
       </property>
      </item>
     </widget>
     <widget class="QLineEdit" name="lineEdit_17">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>40</y>
        <width>171</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="agregarPublicacion">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>180</y>
        <width>171</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Agregar Publicacion</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_18">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>40</y>
        <width>171</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_5">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>110</y>
        <width>171</width>
        <height>28</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Si</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>No</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string/>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>110</y>
        <width>181</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Disponible para prestamo:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="guardarEnArchivo_4">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>510</y>
        <width>171</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Guardar en archivo</string>
      </property>
     </widget>
     <widget class="QPushButton" name="imprimiEstudiantes_4">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>300</y>
        <width>171</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Imprimir</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_4">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>380</y>
        <width>401</width>
        <height>231</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="eliminarPublicacion">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>260</y>
        <width>171</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Eliminar Publicacion</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_16">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>210</y>
        <width>171</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Titulo:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>10</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Year:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>10</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Dato:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>10</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Autor:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>170</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Titulo:</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1163</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
